### DISCUSSION TITLE
Managing Scroll Control in Super Editor

### DISCUSSION THREAD

**Discussion by isaacadariku** [Flutter-Bounty-Hunters/super_editor#2130] [Q&A]
Hi @matthew-carroll @angelosilvestre I’m attaching a component programmatically in Super Editor. When the component reaches the bottom of the page and the keyboard is open on mobile, I need advice on controlling the scroll controller to ensure the new node is visible. We pass the scroll controller to the editor, but all efforts to control it from outside have been unsuccessful. Is there a correct way to do this, or are we missing something? Any insights or suggestions would be greatly appreciated.

Here is a short video showing what we are exploring.

https://github.com/superlistapp/super_editor/assets/46024202/8af3c819-1b55-472e-9133-a1be19151108

**Comment by matthew-carroll** [comment]
> but all efforts to control it from outside have been unsuccessful

On the surface that sounds like a bug. Can you file an issue? Please include a minimal but complete reproduction of the issue so we can take a look.

**Comment by isaacadariku** [comment]
Thanks for the response @matthew-carroll. We have been able to get the scroll controller to work. We were not using the parent controller to scroll the editor. We have since fixed it. Thanks for the help.
